interface/themes/tabs_style_compact.css

```diff
@@ -29,11 +29,12 @@ html
 }
 body
```

Comment:
Did you mean to modify this file or tabs_style_full.css ?

Was hoping to keep Ray's theme intact and just get the switch from div to a working for it.

If want to create another more compact modern theme (like your full), then sounds like we may need to create another theme file and change the names around  (rename Ray's to something else).

As with your full theme there are even more serious mouseover issues happening on this theme (difficulties in dragging the mouse down to the submenu). Have you been able to try your themes from a firefox browser in linux?